substrate/frame/alliance/src/mock.rs

```diff
@@ -52,7 +52,7 @@ parameter_types! {
 #[derive_impl(frame_system::config_preludes::TestDefaultConfig as frame_system::DefaultConfig)]
 impl frame_system::Config for Test {
 	type Block = Block;
-	type AccountData = pallet_balances::AccountData<AccountId>;
+	type AccountData = pallet_balances::AccountData<u64>;
```

Comment:
why's this necessary?

Reply:
@liamaharon
do you mean the change from `AccountId` to `u64`? 
it is explained above: https://github.com/paritytech/polkadot-sdk/pull/1801/files#r1410488971

or do you mean adding override for `type AccountData`? This cannot be removed because `pallet_balances` has  `type Balance = u64;`